docs/iam-usage.rst

```diff
@@ -0,0 +1,177 @@
+IAM User Guide
+===================
+
+.. note::
+  This document assumes basic knowledge of Google Cloud IAM,
+  see `the docs <https://cloud.google.com/iam/docs/>`_ for details
+
+IAM Resources
+-------------
+
+The ``iam`` module provides a number of resources necessary for interacting with IAM.
+
+Members
+~~~~~~~
+An IAM member is one of the following:
+
+- ``iam.user(email)`` an individual Google account
+- ``iam.service_account(email)`` a Google Cloud Service Account
+- ``iam.group(email)`` a Google group.
+- ``iam.domain(domain_name)`` a Google apps domain
+- ``iam.ALL_AUTHENTICATED_USERS`` any authenticated Google user
+- ``iam.ALL_USERS``
```

Comment:
Make a note that `iam.ALL_USERS` means the resource is publicly, anonymous accessible.